packages/css-themes-polyfill/MAINTAINERS.md

```diff
@@ -0,0 +1,72 @@
+# Maintainers
```

Comment:
only one MAINTAINERS.md file needed for a repo. can be removed from package-level.